### DISCUSSION TITLE
App '740' state is 0x202 on Raspberry Pi 4 via Qemu 6.0

### DISCUSSION THREAD

**Discussion by tarach** [GameServerManagers/LinuxGSM#3611] [General]
Hi,
I'm trying to install csgo server on Raspberry Pi 4 ( 8 GB RAM ) but I keep getting:
```
Error! App '740' state is 0x202 after update job.
CWorkThreadPool::~CWorkThreadPool: work complete queue not empty, 1 items discarded.
Failure! Installing csgoserver: SteamCMD: Not enough disk space to download server files
```
I've tried to fix this issue by adding a USB Stick Drive with 32 GB of free space.
After that I've resized qemu drive image with ```qemu-img resize``` command as well as resize the partition with gparted but I keep getting this error. I'm afraid that I'm running out of ideas.
Raspberry Pi: 5.10.63-v8+
Qemu: 6.0.0
Guest System:  Debian 5.10.46-5
![image](https://user-images.githubusercontent.com/351514/136707649-3cc0f78f-4671-40e6-b25d-339bd4e65684.png)

**Comment by johnoclockdk** [comment]
csgo need 10 gb or more to install

**Comment by tarach** [comment]
Than that's even more weird since it has 19G available ... unless it's in the wrong place ...

**Comment by johnoclockdk** [comment]
> Than that's even more weird since it has 19G available ... unless it's in the wrong place ...

it would not run anyway from pi sorry

**Comment by tarach** [comment]
Well I get the idea that it could be really laggy but I would like to try nonetheless. :D 
I've previously hosted game servers on AWS EC2 or a dedicated machines but never tried that on Pi ^^

Either way the disk space issue is really weird and would like to be able to debug it somehow.

**Comment by dgibbs64** [comment]
You might get an old server like cs1.6 to work...maybe but I think you will struggle with CS:GO. I would like to know if you do succeed.
